nengo_loihi/loihi_cx.py

```diff
@@ -382,6 +382,11 @@ def get_loihi(self, seed=None):
     def get_simulator(self, seed=None):
         return CxSimulator(self, seed=seed)
 
+    def validate(self):
+        if len(self.cx_groups) == 0:
+            raise RuntimeError("No neurons marked for execution on-chip. "
```

Comment:
I don't think we've been very judicious about this in Nengo Loihi yet, but we should be using subclasses of `NengoException` in order to make it easy for other tools to catch / deal with exceptions coming from Nengo separate from other exceptions. In this case, I think I would make this a `BuildError`.

Reply:
Sounds good!